src/components/application_manager/include/application_manager/commands/hmi/basic_communication_on_awake_sdl.h

```diff
@@ -29,3 +29,45 @@
  * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
  * POSSIBILITY OF SUCH DAMAGE.
  */
+
+#ifndef SRC_COMPONENTS_APPLICATION_MANAGER_INCLUDE_APPLICATION_MANAGER_COMMANDS_HMI_BASIC_COMMUNICATION_ON_AWAKE_SDL_H_
+#define SRC_COMPONENTS_APPLICATION_MANAGER_INCLUDE_APPLICATION_MANAGER_COMMANDS_HMI_BASIC_COMMUNICATION_ON_AWAKE_SDL_H_
+
+#include "application_manager/commands/hmi/notification_from_hmi.h"
+
+namespace application_manager {
+
+namespace commands {
+
+/**
+ * @brief OnAwakeSDLNotification command class
+ **/
+class OnAwakeSDLNotification : public NotificationFromHMI {
+ public:
+  /**
+   * @brief OnAwakeSDLNotification class constructor
+   * @param message Incoming SmartObject message
+   * @param application_manager reference to ApplicationManager instance
+   **/
+  OnAwakeSDLNotification(const MessageSharedPtr& message,
+                         ApplicationManager& application_manager);
+
+  /**
+   * @brief OnAwakeSDLNotification class destructor
+   **/
+  virtual ~OnAwakeSDLNotification();
+
+  /**
+   * @brief Execute command
+   **/
+  virtual void Run();
```

Comment:
@dev-gh fixed in https://github.com/smartdevicelink/sdl_core/pull/1604/commits/53d637e6c9aad93b3ecd2f432ab2fd36284e14fc